cmd/utils/flags.go

```diff
@@ -315,8 +315,25 @@ var (
 		Category: flags.EthCategory,
 	}
 	OverrideFeynmanFix = &cli.Uint64Flag{
-		Name:     "override.feynmanfix",
-		Usage:    "Manually specify the FeynmanFix fork timestamp, overriding the bundled setting",
+		Name:  "override.feynmanfix",
+		Usage: "Manually specify the FeynmanFix fork timestamp, overriding the bundled setting",
+	}
+	OverrideFullImmutabilityThreshold = &cli.Uint64Flag{
+		Name:     "override.immutabilitythreshold",
+		Usage:    "It is the number of blocks after which a chain segment is considered immutable, only testing purple before Cancun",
```

Comment:
I think we can keep them even after Cancun
just note that it's  only for testing purpose